docs/cmdline-opts/limit-rate.d

```diff
@@ -17,6 +17,9 @@ Appending 'k' or 'K' will count the number as kilobytes, 'm' or 'M' makes it
 megabytes, while 'g' or 'G' makes it gigabytes. The suffixes (k, M, G, T, P)
 are 1024 based. For example 1k is 1024. Examples: 200K, 3m and 1G.
 
+The rate limiting logic works on averaging the transfer speed to no more than
+the set threshold over a period of multiple seconds.
```

Comment:
@fabiankeil What are your thoughts? Is this an accurate description of how the rate limiting works?